testsettings.py

```diff
@@ -118,9 +118,9 @@ def _set_logging_levels(levels):
 REPORTING_DATABASES = {
     'default': 'default',
     'ucr': 'default',
-    'icds-ucr': 'default',
-    'icds-ucr-non-dashboard': 'default',
-    'icds-test-ucr': 'default',
+    'icds-ucr': 'icds-ucr_commcarehq',
```

Comment:
@emord Is this what you had in mind?

Reply:
Your update is indeed what I had in mind. And that's what would be necessary to have the ICDS UCRs in the same database as the aggregate tables. I'm super confused why tests are failing though